app/src/views/FourHundredFour/styles.module.css

```diff
@@ -14,32 +16,41 @@
         flex-direction: column;
         flex-grow: 1;
         gap: var(--go-ui-spacing-2xl);
+       
 
         .top-section {
             display: flex;
+            align-items: center;
             flex-direction: column;
             flex-grow: 1;
             justify-content: center;
             gap: var(--go-ui-spacing-xs);
 
+            .four-hundred-four-heading {
+                background: repeating-linear-gradient(
+                    -50deg,
+                    red 10px,
+                    transparent ,
+                    transparent 20px
+                );
+                -webkit-background-clip: text;
+                -moz-background-clip: text;
+                -webkit-text-fill-color: transparent;
+                font-size: 8rem;
+                font-weight: bold;
```

Comment:
Don't we have a design token for this?

Reply:
We should also define the design tokens for the gradients that's being used here